### DISCUSSION TITLE
Configuring Notifications for Opsgenie

### DISCUSSION THREAD

**Discussion by phillf** [containrrr/watchtower#1687] [Q&A]
Has anyone had any luck with configuring notification being sent to Opsgenie? If yes, how did you configure it?

**Comment by piksel** [comment]
Could you explain what problems you are having? I only know of the person implementing the service in the first place, but it should *just work*.

**Comment by piksel** [comment]
Could you explain what problems you are having? I only know of the person implementing the service in the first place, but it should *just work*.
